engines/bastion_katello/test/subscriptions/subscription-type.directive.test.js

```diff
@@ -40,7 +40,7 @@ describe('Directive: subscriptionType', function() {
     });
 
     it("subscription type Guest when host", function() {
-        $scope.subscription = {virt_only: true, host: {id: 123, name: "hypervisor"}};
+        $scope.subscription = {virt_only: true, hypervisor: {id: 123, name: "hypervisor"}};
```

Comment:
Sorry for the necropost, but this changes the API, no?

Specifically here:

```json
            {
    ...,
    "results": [
        ...,
        {
           ...
            "unmapped_guest": false,
            "virt_only": true,
            "virt_who": true,
            "hypervisor": {
                "id": 25,
                "name": "akofink-desktop.usersys.redhat.com"
            }
        }
    ]
}
```

The "hypervisor" dict used to be "host", right? Is @SatelliteQE aware?

Reply:
Thanks for investigation. We can either make a copy not to break the V2 API or do a documentation about this if there is some kind of technical limitation.